test/data/testTD.py

```diff
@@ -160,7 +160,7 @@ class OrcaROCISTest(GenericTDTest):
     """Customized test for ROCIS"""
     number = 57
     expected_l_max = 2316970.8
-    n_spectra = 9
+    n_spectra = (8, 9)
```

Comment:
Perhaps add a comment why it's a 2-tuple with these values.

Reply:
Oh! This seems a little error prone, especially to one-off errors. It might be more prudent to just override the value for one ORCA version and list that in `testdata`. It won't be a lot of boilerplate, and will also be more clear.